src/app/components/header/header.component.spec.ts

```diff
@@ -185,4 +187,30 @@ describe('HeaderComponent', () => {
         const balanceText = balanceElement.nativeElement.innerText.trim();
         expect(balanceText).toBe('0.75');
     });
+
```

Comment:
I think you don't test if the button for the overlay is actually available depending on how many tokens are listed. Sorry if I misunderstand something here.

Reply:
You are right. I'll add a test!